app.py

```diff
@@ -19,3 +23,43 @@ def get_map():
     return Response(response=map.to_json(),
                     status=200,
                     mimetype="application/json")
+
+
+# API
+valid_moves_example_response = [
+    { # Behaves uniquely. Must choose to do before other moves. This must be remembered between player turns. There can be multiple.
+        'type': 'leech',
+        'player_id': '1234',
+        'amount': 3
+    },
+
+    {
+        'type': 'build',
+        'hex': 11,
+        'navigation': {'QIC': 2, 'BON11': 1},
+        'dig': {'DIG': 3, 'ACT4': 2, 'ACT5': 1}
+    },
+
+    {
+        'type': 'upgrade',
+        'hex': 23,
+        'opt1': 1,
+        'opt2': None
+    },
+
+    {
+        'type': 'action',
+        'bonus-action': 4
+    },
+
+    {
+        'type': 'pass',
+        'tiles': [1, 5, 7, 11]
+    }
+]
+
+
+@api.route('/valid moves')
```

Comment:
Will this work with a space?

Reply:
The route name is `valid moves` which I suspect might not work because it has a space in it.

Reply:
O ya guess swagger fixed it automatically meant to have a -